cmd/podman/machine/rm.go

```diff
@@ -64,7 +64,7 @@ func rm(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	confirmationMessage, remove, err := vm.Remove(vmName, machine.RemoveOptions{})
+	confirmationMessage, remove, err := vm.Remove(vmName, destoryOptions)
```

Comment:
could we change "destoryOptions" to "destroyOptions"?

Reply:
Yes. I will do main update for this to backport here.

Reply:
https://github.com/containers/podman/pull/13562